socorro/external/elasticsearch/supersearch.py

```diff
@@ -463,6 +463,22 @@ def update_field(self, **kwargs):
                 'be created before it can be updated. ' % params['name']
             )
 
+        if 'storage_mapping' in params and old_value['storage_mapping']:
+            print 'remove storage_mapping'
+            # The storage mapping is an object, and thus is treated
+            # differently than other fields by Elasticsearch. If a user
+            # changes it by removing a field from it, that field won't be
```

Comment:
Prefer "changes *the object*"; using "it" twice in a row creates ambiguity, which makes this explanation harder than it should be to parse.  Great comment otherwise though!  Very useful.